Modules/_complex.h

```diff
@@ -0,0 +1,34 @@
+/* Workarounds for buggy complex number arithmetic implementations. */
```

Comment:
I suggest to rename the header file to pycomplex.h, since _complex.h sounds a little bit too generic.